lib/services/android-project-service.ts

```diff
@@ -273,13 +275,13 @@ class AndroidProjectService implements IPlatformProjectService {
 				let libsFolderContents = this.$fs.readDirectory(libsFolderPath).wait();
```

Comment:
is it possible to have jars not on the root level of the lib? As far as I understand from this code, we are taking only the files at the root level